.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = C813
```

Comment:
is it possible to install something like `flake8-config-4catalyzer[py2]` and conditionally trigger rules based on that?